### PR TITLE
[#100] Cleanup UI; multiple works actions; login form bug; custom error page

### PR DESCRIPTION
**Pull request by ponchofiesta**

### CODE REVIEW

File: kitodo-mediaserver-ui/src/main/java/org/kitodo/mediaserver/ui/UiApplication.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 package org.kitodo.mediaserver.ui;
 
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
```
Comment (pbroman) [inline kitodo-mediaserver-ui/src/main/java/org/kitodo/mediaserver/ui/UiApplication.java]: Non-used imports, please remove

**Comment by ponchofiesta** [comment]
OK, I've fixed all that. Please check again.

**Comment by pbroman** [comment]
Show/hide password still don't work, otherwise fine

**Comment by ponchofiesta** [comment]
Sorry, I fixed it in "new" template but not in "edit" template. Now it's fixed.

**Pull request merged**